Makefile

```diff
@@ -15,6 +15,13 @@ endef
 USER_SOURCE ?= rc_user
 
 
+# Map libs variables
+OL_VERSION ?= be12573# September 25 2018 (mind the absence of a space character after the version)
+OL_CESIUM_VERSION ?= 695253277d66a8917fa456f3346c049b88f88eb7 # October 12 2018
+CESIUM_VERSION ?= 54d850855346610fde9b7aa8262a03d27e71c663 # c2c/c2c_patches (Cesium 1.44), April 23 2018
+GEOBLOCKS_LEGACYLIB_VERSION ?= 0101a217be1b7525be8d590910fb8f70295194be # September 24 2018
```

Comment:
What is this `GEOBLOCK` things?

Reply:
Geoblocks is a new vision of how to build applications.
Historically we have big projects with all our code, which ends up as interdependent spaghetti, whatever the design effort we put into it. It also becomes heavy to migrate since everything is should be changed or nothing.

Geoblocks are tiny pieces of code you can compose in a bigger application. At the moment it is still in the alpha stage but I hope I can put it forward.

Regarding this geoblocks/legacylib stuff it allows to do a full build of OpenLayers5 (not just the `@api` symbols).

Reply:
Nice , looks promising